src/transaction.js

```diff
@@ -279,15 +279,15 @@ export default class Transaction {
         return tx;
     }
 
-    buildLeasingTx(public_key, recipient, amount, timestamp) {
+    buildLeasingTx(public_key, recipient, amount, timestamp, fee) {
```

Comment:
As other developers are using this SDK now, the interface change should be compatible with previous versions. Add this line below:
```
fee = typeof fee !== 'undefined' ? fee : Constants.TX_FEE
```

Reply:
Also add adjust fee feature for `buildPaymentTx`, `buildCancelLeasingTx`, `buildRegisterContractTx`, `buildExecuteContractTx` and `buildSendTokenTx`.